// ./genStub echo.proto
// proto --cpp_out=. echo.proto

syntax = "proto3";

package echo;

/*
clientInclude {
    RPCClient.h
    log/Logger.h
    rpc.pb.h
}

serverInclude {
    RPCServer.h
    log/Logger.h
    rpc.pb.h
}

*/

message EchoRequest{
    string message = 1;
}

message EchoResponse{
    string message = 1;
}

service EchoService{
    rpc Echo(EchoRequest) returns (EchoResponse);
}
